### DISCUSSION TITLE
Einrichtung anpassen per Handy Browser?

### DISCUSSION THREAD

**Discussion by Bernd275** [camueller/SmartApplianceEnabler#479] [Q&A]
Hallo in die Runde, gibt es eine Beschränkung von irgendwelchen Browsern die nicht funktionsfähig sind?

Mein Problem ist folgendes, ich hab mir heute morgen gedacht ich passen mal den Zeitplan eines Gerätes an und ich nehme mal das Handy lokal im WLAN dafür. Als Browser hab ich da DuckDuckGo drauf.
Nach dem Speichern reagiert der SAE nicht mehr und zu allem übel sind alle 21 Geräte gelöscht worden, nach einem Hardreset der Stromzufuhr!
Gibt es eine Möglichkeit die gestern noch eingerichteten Geräte wieder zurückzuholen?
Eine Export und Import Funktion für alle Geräte wäre genial.

**Comment by camueller** [comment]
> Mein Problem ist folgendes, ich hab mir heute morgen gedacht ich passen mal den Zeitplan eines Gerätes an und ich nehme mal das Handy lokal im WLAN dafür. Als Browser hab ich da DuckDuckGo drauf.
Nach dem Speichern reagiert der SAE nicht mehr und zu allem übel sind alle 21 Geräte gelöscht worden, nach einem Hardreset der Stromzufuhr!

Ich selbst benutze ausschliesslich Chrome unter Android auf dem Handy, aber es gibt automatisierte Tests für Chrome, Firefox und Safari, in denen alle grundlegenden Funktionen getestet werden. Die Web-Anwendung ist für keinen Browser angepasst, d.h. jeder moderne Browser sollte funktionieren.

> Gibt es eine Möglichkeit die gestern noch eingerichteten Geräte wieder zurückzuholen?

Klar. Einfach die Backups der beiden XML-Dateien zurückspielen, die Du hoffentlich hast.

> Eine Export und Import Funktion für alle Geräte wäre genial.

Nicht geplant, weil Backup ausreicht.

**Reply by Bernd275** [reply]
Hätte ich irgendwann mal etwas von Backup gelesen, dann hät ich auch ein Backup ;-) hab's unter Konfigurationsdateien gefunden
